src/Symfony/Component/Validator/Constraints/AtLeastOneOfValidator.php

```diff
@@ -42,9 +42,11 @@ public function validate($value, Constraint $constraint)
                 continue;
             }
 
+            $context = $this->context;
             $executionContext = clone $this->context;
             $executionContext->setNode($value, $this->context->getObject(), $this->context->getMetadata(), $this->context->getPropertyPath());
             $violations = $validator->inContext($executionContext)->validate($value, $item, $this->context->getGroup())->getViolations();
+            $this->context = $context;
```

Comment:
just wondering: would it make sense to nest this in a finally block?

Reply:
I don't think so. First, we don't do that anywhere else and second `validate()` is not supposed to throw. If it did, I would expect more things to no longer work as expected.